.gitlab-ci.yml

```diff
@@ -69,13 +67,15 @@ before_script:
     - set -e
     - make -f Makefile.dune world
     - set +e
+    - tar cvfj _build.tar.bz2 _build
   variables:
     OPAM_SWITCH: edge
     OPAM_VARIANT: "+flambda"
   artifacts:
     name: "$CI_JOB_NAME"
     paths:
-      - _build/
+      - _build/log
+      - _build.tar.bz2
```

Comment:
Seems reasonable. Why the separate /log?

Reply:
So it is browsable from the interface?

Reply:
ok